### ISSUE TITLE
Benchmark: dual path network with wide resnet 28-10 as backbone

### ISSUE THREAD

**Issue by Queequeg92** [zalandoresearch/fashion-mnist#42]
Classifier: dual path network with WideResNet28-10 as the backbone network (47.75M).
Preprocessing: standard preprocessing (mean/std subtraction/division) and augmentation (random crops/horizontal flips).
Fashion test accuracy: 95.73%
Code:https://github.com/Queequeg92/DualPathNet
References:
[1] Chen, Yunpeng, Jianan Li, Huaxin Xiao, Xiaojie Jin, Shuicheng Yan, and Jiashi Feng. "Dual Path Networks." arXiv preprint arXiv:1707.01629 (2017).
[2] Zagoruyko, Sergey, and Nikos Komodakis. "Wide residual networks." arXiv preprint arXiv:1605.07146 (2016).

**Comment by hanxiao** [comment]
Many thanks for your contribution and the links to the paper. You are welcome include experiments on fashion-mnist in your final publication.

**Comment by Queequeg92** [comment]
It's just an implementation for fun. I did not fine-tune hyper-parameters. Thanks for your awesome dataset.
